docs/docs/guides/automate/sensors/using-sensors.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Using sensors in projects
+sidebar_position: 250
+description: Using sensors in Dagster dg projects for entities such as assets and jobs.
+---
+
+import DgComponentsRc from '@site/docs/partials/\_DgComponentsRc.md';
+
+<DgComponentsRc />
+
+:::note Prerequisites
+
+Before following this guide, you will need to [create a project](/guides/build/projects/creating-a-new-project) with the [`create-dagster` CLI](/api/dg/create-dagster).
+
+:::
+
+[Assets](/guides/build/assets) and [jobs](/guides/build/jobs) frequently use sensors that are instantiated elsewhere in the project.
+
+For example, if you have created a new Dagster project with `dg` called `my_project`, you can define the sensors at `src/my_project/defs/sensors.py`:
+
+Sensor binding can happen at any level of the `defs` hierarchy. If you moved `asset_one` in this example to a subdirectory, you could leave the existing `sensors.py` file at `src/defs/sensors.py`:
+
+```
+src
+└── my_project
+    └── defs
+        ├── assets
+        │   └── asset_one.py # contains def asset_one():
+        └── sensors.py # contains dg.sensor
+```
+
+## Scaffolding sensors
+
+To create a sensor dictionary like the above, you can run the following:
+
+```bash
+dg scaffold defs dagster.sensor path/to/sensors.py
```

Comment:
While this is technically true, the outcome below comes from `dg scaffold defs dagster.sensor sensors.py`, so I recommend using that instead, plus a link to the dg CLI docs for more info on scaffolding sensors (which will be in place once Ben adds more CLI docs).